src/components/WebexParticipantRoster/WebexParticipantRoster.jsx

```diff
@@ -5,15 +5,23 @@ import {List} from '@momentum-ui/react';
 import WebexParticipant from '../WebexParticipant/WebexParticipant';
 import useMemberships from '../hooks/useMemberships';
 
+/**
```

Comment:
Can you add the TODO comment so it's easy to find?
```suggestion
// TODO: Figure out how to import JS Doc definitions and remove duplication.
/**
```

Reply:
I did find that you can do an empty import and VSCode will recognize it. Should we create a `types.js`?

https://github.com/jsdoc/jsdoc/issues/1537

Reply:
yeah, I saw that too but I deferred to see if there's another way